mautic/mautic
# Click tracking stop working

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      |
| Related developer documentation PR URL |
| Issue(s) addressed                     |




#### Description:

This is regression issue from https://github.com/mautic/mautic/pull/9265
Clicks from email was not count



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create email with link
3. Try click on link
4. Click was not count
5. After PR should work like before
